lib/puppet/type/cisco_bgp_neighbor_af.rb

```diff
@@ -387,6 +389,24 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end
 
+  newproperty(:route_map_in) do
+    desc 'route-map in state. Valid values are a string defining the name ' \
```

Comment:
No. I did not miss your point, but I did incorrectly assume that route-map could *NOT* be an integer :).  Looks good.

:+1: